.github/workflows/node.js.yml

```diff
@@ -53,12 +53,7 @@ jobs:
         DB_USER: 'postgres'
         DB_DATABASE: 'postgres'
         DB_PASSWORD: 'postgres'
-        # Entry points for crawler, in a comma separated list
-        ENTRIES: 's2.ripple.com,s.altnet.rippletest.net,s.devnet.rippletest.net'
         # Rippled Server with access to manifest command
         RIPPLED_RPC_ADMIN: 'https://xrpl.ws/'
-        # Validator lists for main, dev, and test networks
-        VL_MAIN: 'vl.ripple.com'
-        VL_TEST: 'vl.altnet.rippletest.net'
-        VL_DEV: 'vl.devnet.rippletest.net'
-  
\ No newline at end of file
+        # (Placeholder) Mainnet entry point for crawler (without https://).
```

Comment:
Why is this a placeholder? (What process updates it to a non-placeholder value?)

Reply:
It's a placeholder because something needs to be there so the tests will run properly, but it doesn't actually need to be anything valid.